### DISCUSSION TITLE
Feature Request: Release scripts as versioned NPM Packages

### DISCUSSION THREAD

**Discussion by AdamCoulterOz** [Azure/api-management-developer-portal#2639] [General]
Can the scripts be released on npm as versioned standalone packages so pipelines that deploy portal customisations can reference it? Ideally, refactored as a proper CLI, but even the standalone scripts would be helpful.

One package... `aipm-devportal` or whatever, but with two supported versions... `2.x` and `3.x`, i.e. not both folders in the same package.

**Comment by ghost** [comment]
@AdamCoulterOz, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
@mikebudzynski, can you please comment on this?

**Comment by AdamCoulterOz** [comment]
Ultimately as part of the az cli would be ideal, but I'm not going to hope for that too much.
